src/panel.tsx

```diff
@@ -14,7 +14,7 @@ const Iframe = styled.iframe({
 });
 
 function parseShareURL(url: string): string | void {
-  return url.split("share.goabstract.com/")[1];
+  return url.split(/share.(?:go)?abstract.com\//)[1];
 }
```

Comment:
`inferShareId` was copied from [helpers in the sdk](https://github.com/goabstract/abstract-sdk/blob/e18353108156025614029050e46566151c93ec0f/src/util/helpers.js#L4). Looks like it would still be outdated there too

Would be nice to update the sdk an expose it as public api